compare-metrics.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
```

Comment:
nit: Since the scripts are so related to each other, can you merge them in a single script and decide the operation (compute or compare) with an argument?